css/style.css

```diff
@@ -134,14 +136,41 @@ body {
 	position: absolute;
 	top: 0;
 	text-align: center;
+	z-index: 1;
 }
 
 .slides {
 	display: none;
 }
 
-.slides img {
-	color: #fff;
```

Comment:
The images in the images slider were done differently than expected before. I put them in the css instead of the html so there is a big change here